mmv1/third_party/terraform/website/docs/d/compute_region_network_endpoint_group.markdown

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: "Compute Engine"
+page_title: "Google: google_compute_region_network_endpoint_group"
+description: |-
+  Get access to a Network Endpoint Group's Attributes.
+---
+
+# google\_compute\_region\_network\_endpoint\_group
+
+Use this data source to access a Regional Network Endpoint Group's attributes.
+
+The NEG may be found by providing either a `self_link`, or a `name` and a `region`.
+
+## Example Usage
+
+```hcl
+// Cloud Run Example
+resource "google_compute_region_network_endpoint_group" "cloudrun_neg" {
```

Comment:
Since this isn't the example used in the test, how was this tested? did you manually run it?

Reply:
@googlerjk this question still applies